src/main/java/User.java

```diff
@@ -0,0 +1,28 @@
+import java.util.ArrayList;
+
+/**
+ * Created by vasantia on 7/11/16.
+ */
+public class User {
+
+    private String name;
+    private String password;
+    ArrayList<Message> myMessages = new ArrayList<>();
```

Comment:
Don't forget to program to the interface.  `List<Message> myMessages = new ArrayList<>();`